### DISCUSSION TITLE
Integration of Laravel Pulse

### DISCUSSION THREAD

**Discussion by jbelien** [spatie/laravel-prometheus#28] [Ideas]
End of last year, Laravel Pulse has been announced: https://pulse.laravel.com/ (see https://laravel-news.com/announcing-laravel-pulse).

I think it would be awesome if those metrics could also sent to Prometheus using your library.

Is that something you would consider ?
Thanks!

**Comment by freekmurze** [comment]
I'd welcome a PR that adds this!
